src/odfesqlodbc/es_statement.cpp

```diff
@@ -140,6 +140,9 @@ RETCODE ExecuteStatement(StatementClass *stmt, BOOL commit) {
         // Assign directly
         SC_set_Result(stmt, res);
     }
+    if (commit) {
```

Comment:
I don't understand why having a flag called commit would trigger getting another result set? Comments? Or rename the variable?

Reply:
We are getting metadata and rows from elasticsearch when we send a query, When user calls SQLPrepare, we are not committing datarows but In the case of SQLExecDirect, we are committing datarows in this function. I will add a comment here.

Reply:
I'm unsure what commit means here - commit generally means in the context of a transaction, but it's not clear why a commit would mean get the next result?

Reply:
The basic function performed by commit here is deciding whether to add datarows in QResultClass or keep it in the queue to retrieve later.
I am not entirely sure what was the reason for using name `commit` but any other suggestion is welcome.